packages/hub/db/migrations/20230313142122856_add-memo-to-scheduled-payments.ts

```diff
@@ -0,0 +1,12 @@
+import { MigrationBuilder } from 'node-pg-migrate';
+
+const TABLE = 'scheduled_payments';
+const PRIVATE_MEMO = 'private_memo';
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.addColumn(TABLE, { [PRIVATE_MEMO]: { type: 'string', notNull: false, default: undefined } });
```

Comment:
Perhaps `notNull: false, default: undefined` might not be needed? It sounds like it's default.